mllib/src/main/scala/org/apache/spark/ml/tuning/TrainValidationSplit.scala

```diff
@@ -177,7 +202,9 @@ class TrainValidationSplit @Since("1.5.0") (@Since("1.5.0") override val uid: St
   }
 
   @Since("2.0.0")
-  override def write: MLWriter = new TrainValidationSplit.TrainValidationSplitWriter(this)
+  override def write: TrainValidationSplit.TrainValidationSplitWriter = {
```

Comment:
Was this meant to be for TrainValidationSplitModel, not the Estimator?

Reply:
Ah, there is two `write` method, one for `Estimator` and another for `Model`.
We only need to change the return type of `write` method for `model`.